rules/clean_wp3_outbox_from_TE_runs.yaml

```
@@ -0,0 +1,17 @@
+---
+name: "clean_TE_outbox"
+pack: "ductus_clean"
+description: "Rule that triggers cleaning of outbox data for TE"
+enabled: false
+
+trigger:
+  type: "core.st2.CronTimer"
+  parameters:
+    timezone: "UTC"
+    day_of_week: 5 # or day_of_week: "sun"
+    hour: 0
```

maybe run this one a bit after TE inbox?
```suggestion
    hour: 1
```